sormas-backend/src/test/resources/testcontainers/setup_sormas.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I would call this file `setup_sormas_db` to state that it sets up only the database not a whole sormas app